diary/logdb.py

```diff
@@ -69,6 +69,10 @@ def close(self):
         except sqlite3.ProgrammingError:
             pass
 
+    def destroy(self):
```

Comment:
Very misguided. I'm incredibly happy to see you contributing and I like the changes you have made. This is my first open source project and I'm trying to prioritize some things. Regardless, I'll try to be more responsive and assist you.